2-mandatory/4-space-colonies.js

```diff
@@ -15,7 +15,15 @@
   
 */
 
-function getSettlers() {}
+function getSettlers(familiesArray) {
```

Comment:
great solution, but you can simplify it by using ``filter()``. Can you modify it?